packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -513,4 +513,36 @@ void main() {
 
     semantics.dispose();
   });
+
+  testWidgets('Increased/decreased values are annotated', (WidgetTester tester) async {
```

Comment:
If it's possible to test, it would be good to test that some of the complex assertions are in place. For example, trying to give null to increasedValue, or supply one and not the other, etc.